src/generation.rs

```diff
@@ -107,6 +108,9 @@ impl Template {
             })
             .filter_map(|e| e.ok());
 
+        // Register the `slug` filter
+        Tera::default().register_filter("slugify", |value, _| slugify(value));
```

Comment:
That's not how you register it, this is dropping the Tera instance right away. Since the code uses Tera::one_off a few times, it's probably better to have a function to render a template that looks like

```rust
fn render_template(content, context) -> Result<String> {
  let mut tera = Tera::default();
  tera.register_filter("slugify", |value, _| slugify(value));
  tera.add_raw_template("one_off", content)?;
  let res = tera.render("one_off", &context).map_err(|err| new_error(ErrorKind::Tera { err, path: None }))?;
  Ok(res)
}
```

or something similar

Reply:
Thanks for the insight.

I thought for a second that `Tera::default()` was referring to a static instance.